dumper-integration-tests/redshift-tests/src/main/java/com/google/edwmigration/dumper/sql/SqlUtil.java

```diff
@@ -32,23 +31,21 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/**
- * A helper class for reading .sql files.
- */
+/** A helper class for reading .sql files. */
 public final class SqlUtil {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(SqlUtil.class);
 
-  private SqlUtil() {
-  }
+  private SqlUtil() {}
 
   /**
    * @param sqlPath Path to an .sql file.
    * @return File contents, never null.
    */
   public static String getSql(String sqlPath) {
     try {
-      return readFileToString(new File(sqlPath), StandardCharsets.UTF_8);
+      URL url = Resources.getResource(sqlPath);
+      return Resources.toString(url, StandardCharsets.UTF_8);
     } catch (IOException exception) {
       throw new UncheckedIOException(format("Error while reading sql file %s", sqlPath), exception);
```

Comment:
Can we please avoid unchecked exceptions?

Reply:
Done.